MovingBlocks/Terasology
# chore: removed count field for issue #4242

### Contains

Fixes #4242 providing a lighter version of `AfkAuthoritySystem.java`.

- [x] Removed `count` field from `AfkAuthoritySystem.java`

- [x]  Compared the behavior of the `[afk]` tag in a multiplayer context before and after `count` removal, found no differences.

### How to test

- Start/Join a multiplayer game
- Type `afk` in the game console 
- Check the `[afk]` tag in the players list, move your character and check again
